trimmer_decorator.rb

```diff
@@ -0,0 +1,8 @@
+require './decorator'
+
+# A class to trim names
+class TrimmerDecorator < Decorator
+  def correct_name
+    super[0..9]
+  end
```

Comment:
- [ ] For this method, it is expected to trim the name passed if it has above 10 characters or else return the name as is. The name has a maximum of 10 characters. If it's longer it should trim the word. Let's implement that 👍🏽

```ruby
    def correct_name
      @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
    end
```